packages/mapviewer/package.json

```diff
@@ -116,6 +116,7 @@
         "vite-node": "catalog:",
         "vite-plugin-static-copy": "catalog:",
         "vitest": "^3.0.5",
+        "vite-plugin-vue-devtools": "^7.7.1",
```

Comment:
can you move this version definition to the catalog and use `"catalog:" here instead?

Reply:
:+1: :heavy_check_mark: